apps/docs/content/guides/functions/unit-test.mdx

```diff
@@ -40,11 +40,12 @@ import {
   assertEquals,
 } from 'https://deno.land/std@0.192.0/testing/asserts.ts'
 import { createClient, SupabaseClient } from 'https://esm.sh/@supabase/supabase-js@2.23.0'
-import { delay } from 'https://deno.land/x/delay@v0.2.0/mod.ts'
+import { load } from 'https://deno.land/std@0.224.0/dotenv/mod.ts'
 
 // Set up the configuration for the Supabase client
-const supabaseUrl = Deno.env.get('SUPABASE_URL') ?? ''
-const supabaseKey = Deno.env.get('SUPABASE_ANON_KEY') ?? ''
+const env = await load()
+const supabaseUrl = env['SUPABASE_URL'] ?? ''
+const supabaseKey = env['SUPABASE_ANON_KEY'] ?? ''
```

Comment:
I needed to use load to get Environment variables.
Ref: https://docs.deno.com/runtime/manual/basics/env_variables#env-file